vllm/entrypoints/api_server.py

```diff
@@ -1,7 +1,10 @@
 import argparse
 import json
 from typing import AsyncGenerator
-
+import torch
+if torch.version.cuda is None and torch.version.hip is None:
```

Comment:
@kzawora-intel @mdvoretc-intel 
is it the right way to import habana-specific packages?